src/test/java/org/fcrepo/client/integration/FcrepoClientIT.java

```diff
@@ -501,6 +504,28 @@ public void testGetDisableRedirects() throws Exception {
         assertEquals(externalURI, getResponse.getLocation());
     }
 
+    @Test
+    public void testGetWantDigest() throws Exception {
+        // Creating a binary for retrieval
+        final String mimetype = "text/plain";
+        final String bodyContent = "Hello world";
+        final FcrepoResponse response = client.post(new URI(serverAddress))
+                .body(new ByteArrayInputStream(bodyContent.getBytes()), mimetype)
+                .perform();
+
+        final URI url = response.getLocation();
+
+        // Get the content of the object after the first 6 bytes
```

Comment:
"after the first 6 bytes"? 
Can you explain?

Reply:
Um, yeah, copy and paste from another test :) it should be more relevant now